esignet-service/make.sh

```
@@ -0,0 +1,277 @@
+  sed -i.bak "s|replace github.com/thunder-id/thunderid => $THUNDER_MODULE .*|replace github.com/thunder-id/thunderid => $THUNDER_MODULE $version|" go.mod
+  rm -f go.mod.bak
+  echo "go.mod updated"
+}
+
+target_tidy() { ## Run go mod tidy
+  go mod tidy
+}
+
+target_clean() { ## Remove build artefacts and coverage output
+  rm -f "$BINARY" engine
+  rm -rf bin/ "$OUT_DIR/"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)